test/mi-mctp.c

```diff
@@ -713,6 +741,655 @@ static void test_mpr_mprt_zero(nvme_mi_ep_t ep, struct test_peer *peer)
 	assert(rc == 0);
 }
 
+enum AemEnableState {
+	AemEnableGetEnabled,
+	AemSetToDisabled,
+	AemEnableSetEnabled,
+	AemProcess,
+	AemAckResponse,
+	AemAckReceived
+};
+
+enum AemFailureCondition {
+	AemFcNone,
+	AemFcBadGetConfigHeaderLen,
+	AemFcBadGetConfigTotalLen,
+	AemFcBadGetConfigBufferLen,
+	AemFcBadOccRspHdrLen_Sync,
+	AemFcBadOccRspTotalLen_Sync,
+	AemFcBadOccRspBufferLen_Sync,
+	AemFcBadOccRspHdrLen_Aem,
+	AemFcBadOccRspTotalLen_Aem,
+	AemFcBadOccRspBufferLen_Aem,
+};
```

Comment:
These are a little inconsistent with existing coding style; if this is to match up with some other definitions we can probably let it slide, being test code. If not though, can you make these consistent with the rest here? mainly just `snake_case`.